playwright/e2e/resize.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expect, test } from '../support/test-helpers';
+
+test.describe('resize and pinning', () => {
+  const example = 'pinning';
+  test(example + ' resize column', async ({ si, page }) => {
+    await si.visitExample(example);
+
+    const nameColumn = page.getByRole('columnheader', { name: 'Name' });
+    const genderColumn = page.getByRole('columnheader', { name: 'Gender' });
+    const cityColumn = page.getByRole('columnheader', { name: 'City' });
+    const stateColumn = page.getByRole('columnheader', { name: 'State' });
+
+    const originalWidth = await nameColumn.boundingBox();
+
+    await expect(nameColumn).toHaveAttribute('resizeable');
+    await expect(genderColumn).toHaveAttribute('resizeable');
+    await expect(cityColumn).toHaveAttribute('resizeable');
+    await expect(stateColumn).toHaveAttribute('resizeable');
+
+    await si.runVisualAndA11yTests('check-resize-attribute');
+
+    // Resize name column
+    const resizeHandler = nameColumn.locator('span[class="resize-handle"]');
+    const originBoundingBox = await resizeHandler.boundingBox();
+    const increaseWidthBy = 300;
+
+    await page.mouse.click(originBoundingBox.x, originBoundingBox.y);
+    await page.mouse.down();
+    await page.mouse.click(originBoundingBox.x + increaseWidthBy, originBoundingBox.y);
+    await page.mouse.up();
+
+    const updatedWidth = await nameColumn.boundingBox();
+
+    expect(updatedWidth.width).toBe(originalWidth.width + increaseWidthBy);
+
+    await si.runVisualAndA11yTests('resize-name-column');
+  });
+
+  test(example + ' pinning column', async ({ si, page }) => {
```

Comment:
This test is capturing multiple cell text in browser selection mode.

<img width="578" alt="image" src="https://github.com/user-attachments/assets/f188a0d4-b89f-47ba-af4e-5f73c3dc06b1" />